src/BladeInstance.php

```diff
@@ -60,13 +60,16 @@ class BladeInstance implements BladeInterface
     /**
      * Create a new instance of the blade view factory.
      *
-     * @param string $path The default path for views
+     * @param string|array $path The default path for views
      * @param string $cache The default path for cached php
      * @param DirectivesInterface $directives
      */
-    public function __construct(string $path, string $cache, DirectivesInterface $directives = null)
+    public function __construct($path, string $cache, DirectivesInterface $directives = null)
     {
-        $this->path = $path;
+        if (!is_array($path)) {
```

Comment:
I'm not a fan of magic type juggling like this, accidentally passing the wrong type tends to get masked and presented as a hard to understand error message.

Maybe a static method to create with multiple paths, or maybe we could had an `addPath()` method, let me give it some thought